docs/enterprise/metrics-app-overview-page.md

```diff
@@ -72,9 +59,55 @@ The CU over time chart displays the following elements:
 
 * **CU % Limit** - A grey dotted line that shows the threshold of the allowed percent of CU consumption for the selected capacity. Columns that stretch above this line, represent timepoints where the capacity is overloaded.
 
+
+### Throttling
+
+Displays the Interactive delay, Interactive rejection and Background rejection over time. Use the tabs at the top of the visual to toggle between Interactive delay, Interactive rejection and Background rejection and use tabs from the top right corner of the visual to toggle how the visual is displayed. Timepoints with above 100% value are rendered with darker color. 
+
+The throttling chart displays the following elements:
+
+  * **Interactive delay** - Billable interactive Operations are delayed if the value is above 100%.
```

Comment:
we are ok to discuss the actual policies that back these changes.  They have been published as of 9/18 at : 
https://blog.fabric.microsoft.com/en/blog/fabric-capacities-everything-you-need-to-know-about-whats-new-and-whats-coming?ft=Chris%20Novak:author


<img width="519" alt="throttling policies" src="https://github.com/MicrosoftDocs/fabric-docs/assets/110125595/b62eb364-d649-4893-a275-ad7a49e91e8f">